### ISSUE TITLE
Choose font

### ISSUE THREAD

**Issue by battle-panda** [pandacorps/twist-design#4]
Options:
- [Roboto](https://fonts.google.com/specimen/Roboto?category=Sans+Serif,Display#standard-styles)
  - Variability in thickness
  - Humanist approach
  - Designed by Google
- [Noto Sans](https://fonts.google.com/specimen/Noto+Sans#standard-styles)
  - Goal to be supported in major languages with comparable sizing
  - Humanist approach
  - Basic regular, italic, bold, and bold italic
  - Designed by Google
- [Raleway](https://fonts.google.com/specimen/Raleway)
  - Several options
  - Multiple designers

**Comment by battle-panda** [comment]
@compiledpanda After looking through fonts on Google Fonts, I would propose either Roboto or Noto Sans. I did look at Raleway, but I found it less readable. I'm leaning a little toward Noto Sans, only because I am getting a little tired of Roboto. Noto Sans has better language support, but I don't think that should be a primary determining factor. Roboto is a good font and has support for a wide range of thicknesses, so if you feel strongly about it, I'm not going to fight too hard for Noto Sans.
